cmd/template/main.go

```diff
@@ -113,6 +113,14 @@ tmp_dir = "tmp"
         `)
 }
 
+func EnvFileTemplate() []byte {
+  return []byte(`#Environment Variables
+  
+# Port to run the application on
+PORT="8080"
+  `)
+}
+
```

Comment:
This is great! However, we have another [PR](https://github.com/Melkeydev/go-blueprint/pull/51)
That is moving go-blueprint away from this style and using tmpl + embed more.
Can we make that change here as well?
Also, we should add a .env ignore in the .gitignore generator as well in this PR!

Reply:
good call, I've re-based and implemented as embed, also add `.env` to gitignore